code/modules/materials/definitions/solids/materials_solid_stone.dm

```diff
@@ -32,6 +32,7 @@
 	uid = "solid_sandstone"
 	lore_text = "A clastic sedimentary rock. The cost of boosting it to orbit is almost universally much higher than the actual value of the material."
 	value = 1.5
+	melting_point = T0C + 400
```

Comment:
It seems that silicon dioxide (quartz, the main component of sandstone) melts at 1,713C? Which is a bit high. It looks like heating it to 600C might cause it to start fracturing so we could probably use that as a limit.

Reply:
Made it 600